modules/nw-egress-ips-about.adoc

```diff
@@ -1,11 +1,8 @@
 // Module included in the following assemblies:
 //
-// * networking/openshift_sdn/assigning-egress-ips.adoc
 // * networking/ovn_kubernetes_network_provider/configuring-egress-ips-ovn.adoc
 
-ifeval::["{context}" == "egress-ips"]
```

Comment:
Same point here, make sure to remove the corresponding ifeval block at the end of the file.

Reply:
I _think_ I got all of this taken out.